io.github.prateekmedia.appimagepool.yml

```diff
@@ -14,6 +14,7 @@ finish-args:
   - --socket=pulseaudio
   - --share=network
   - --device=dri
+  - --allow=devel
```

Comment:
This shouldn't be needed. The app should use `flatpak-spawn --host` to execute binaries when running as Flatpak.